README.md

```diff
@@ -8,7 +8,7 @@ With the website running, follow the steps below:
 
 1. Visit [the website](https://aathish04.github.io/moodlearchiver) and enter your credentials using the form provided.
 
-    This website was only tested with the LMS of SSN College of Engineering, but you can enter your own institution's Moodle Backend, and it should still work.
+    This website was only tested with the LMS of SSN College of Engineering/SNU Chennai, but you can enter your own institution's Moodle Backend, and it should still work.
```

Comment:
The line should read "... the LMSes of SSN College of Engineering and Shiv Nadar University Chennai". People already get confused between the institutions and I'm not going to contribute to that problem 😆